MetaMorpheus/EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -16,23 +16,32 @@
 using System.Threading.Tasks;
 using Omics.Modifications;
 using Omics;
+using Easy.Common.Extensions;
 
 namespace EngineLayer
 {
-    public static class PEP_Analysis_Cross_Validation
+    public class PepAnalysisEngine
```

Comment:
this is still referred to as PEPValueAnalysisGeneric.cs at a high level. I feel like that should be changed to PepAnalysisEngine